### DISCUSSION TITLE
Consulo Discord Channel

### DISCUSSION THREAD

**Discussion by TCROC** [consulo/consulo#441] [Ideas] [answered]
A lot of the software that we use for development such as [Mirror](https://github.com/vis2k/Mirror) have Discord channels that the community engages with each other on.  Does Consulo have a Discord channel to join?  If not, could we create one?  This might help grow use and encourage development from others on the IDE if we all have a place to communicate with each other.

**Comment by TCROC** [comment, marked as answer]
Here's an example of the Mirror discord server: https://discord.gg/NmjSAdqWp3

**Reply by TCROC** [reply]
And here Discord in case you aren't familiar: https://discord.com/

**Reply by VISTALL** [reply]
I'm already have discord, we talk about it in another thread. https://discord.gg/Ab3Ka5gTFv

Since community too small, I'm idk need or not. I don't known that channel need:)

**Reply by TCROC** [reply]
Awesome!  Can we add that link to the [README](https://github.com/consulo/consulo/blob/master/README.md) at the main repo?  We could also maybe add it under the "Help" tab in Consulo.

![image](https://user-images.githubusercontent.com/24307049/101262501-50861880-370d-11eb-82fa-96fd7aff240f.png)

Along with maybe adding a "Thanks for Downloading Consulo" with the link in it for first time downloads.  Just some ideas for growing the Discord community.

**Reply by VISTALL** [reply]
Added both suggestions. Thanks
